client/server/ClassVisitor.js

```diff
@@ -36,17 +36,35 @@ class ClassVisitor {
                 if (!this.completionItems.find(function (elem, index, self) {
                     return elem.data === `id_${yidContext.text}_${currentClassId}`;
                 })) {
+                    const documentation = this.getDocumentation(node.field());
                     this.completionItems.push({
                         label: `${yidContext.text}`,
                         kind: vscode_languageserver_1.CompletionItemKind.Property,
                         data: `id_${yidContext.text}_${currentClassId}`,
-                        detail: `Attribute of class ${currentClassId}.`
-                        //,documentation: "Its documentation can come from predefinedObjects.xml"
+                        detail: `Attribute of class ${currentClassId}.`,
+                        documentation: `${documentation}`
                     });
                 }
             }
         }
     }
+    getDocumentation(options) {
+        try {
+            for (const option of options) {
```

Comment:
General remark: I avoid having, in a given scope, two variable names that only differ by a trailing “s”. In such cases as this one, I generally resort to using names such as “oneFoo” or “aFoo”, and sometimes even “aSingleFoo”.

Reply:
I don't even know what those “options” are. o( +_+ )o Shame on me.
Any help? Comments? Whatever?